next.clayui.com/package.json

```diff
@@ -4,10 +4,11 @@
   "version": "3.0.0",
   "license": "MIT",
   "scripts": {
-    "copy:clay-css": "yarn copy:clay-css-css && yarn copy:clay-css-icons && yarn copy:clay-css-js",
+    "copy:clay-css": "yarn copy:clay-css-css && yarn copy:clay-css-icons && yarn copy:clay-css-js && yarn copy:clay-css-site-images",
     "copy:clay-css-css": "cpx '../node_modules/clay-css/lib/css/*.{css,css.map}' static/css",
     "copy:clay-css-icons": "cpx '../node_modules/clay-css/lib/images/icons/*.svg' static/images/icons",
     "copy:clay-css-js": "cpx '../node_modules/clay-css/lib/js/*.{js,js.map}' static/js",
+    "copy:clay-css-site-images": "cpx '../node_modules/clay-css/src/content/site-images/*.svg' static/images",
```

Comment:
hey @diegonvs we need this task here? Probably this will not change at all times and we also will not always need all the images that are added to this folder. If not we will be running this at all times.